src/NServiceBus.Callbacks/CallbacksSupport.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using NServiceBus.Callbacks;
+
+    class CallbacksSupport : Feature
```

Comment:
We need to use the string name in the sql and rabbit transport

```
if(Feature.IsEnabled("CallbackSupport "))
{
   EnableTheCallbackQueueReceiver();
} 
```

Not a spelling expert but isn't the`trailing `s` in CallbacksSupport wrong?